main/realmain.cc

```diff
@@ -358,7 +357,7 @@ void runAutogen(const core::GlobalState &gs, options::Options &opts, const autog
 
 int realmain(int argc, char *argv[]) {
 #ifndef SORBET_REALMAIN_MIN
-    absl::InitializeSymbolizer(argv[0]);
+    initializeSymbolizer(argv[0]);
```

Comment:
Is there some way we could namespace this (e.g. `sorbet::initializeSymbolizer`) or otherwise disambiguate it from `absl::InitializeSymbolizer`? Just thinking it might be nice to make it clear to folks who are reading this code and happen to be familiar with `absl::InitializeSymbolizer` that we're not directly calling the `absl` function here.

Reply:
Yeah i thought about that, but i noticed that none of our os.cc functions do any namespacing. I'm not opposed to it if you want to take a stab at it, but it seems to work as is, so i'm inclined to not put further work into it.